_includes/releases/v22.2/v22.2.6.md

```diff
@@ -0,0 +1,127 @@
+## v22.2.6
+
+Release Date: March 3, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-6-security-updates">Security updates</h3>
+
+- Introduced the `server.user_login.downgrade_scram_stored_passwords_to_bcrypt.enabled` [cluster setting](../v22.2/cluster-settings.html) , which defaults to `true`. If it is `true` **and** `server.user_login.password_encryption` is `crdb-bcrypt`, then during login, the stored hashed password will be converted from [`SCRAM`](../v22.2/security-reference/scram-authentication.html) to `bcrypt`. Previously, the process to change passwords from `SCRAM` hashing to `bcrypt` hashing was always manual and took a long time. An operator may wish to use this cluster setting if their tools do not support `SCRAM`. [#97628][#97628]
+
+<h3 id="v22-2-6-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Fixed a bug in changefeeds where long running [initial scans](../v22.2/create-changefeed.html#initial-scan) will fail to generate checkpoints. Failure to generate [checkpoints](../v22.2/change-data-capture-overview.html#how-does-an-enterprise-changefeed-work) is particularly bad if the changefeed restarts. Without checkpoints, the changefeed will restart from the beginning, and in the worst case, when exporting substantially sized tables, the changefeed initial scan may not complete. [#97049][#97049]
+- Added support for an optional external ID when [assuming a role](../v22.2/cloud-storage-authentication.html). You can use this by extending the `ASSUME_ROLE` parameter to the format `ASSUME_ROLE={role};external_is={id}`. When using [role chaining](../v22.2/cloud-storage-authentication.html#amazon-s3-role-chaining), you can associate each role in the chain with a different external ID. For example:
+
+    ~~~sql
+    ASSUME_ROLE={roleA};external_id={idA},{roleB};external_id=<idB>,{roleC}
+    ~~~
+
+    This will use external ID `{idA}` to assume delegate `{roleA}`, then use external ID `{idB}` to assume delegate `{roleB}`, and finally no external ID to assume the final role `{roleC}`. [#96531][#96531]
+- Changefeeds no longer require the `COCKROACH_EXPERIMENTAL_ENABLE_PER_CHANGEFEED_METRICS` environment variable in order to use [`metrics_label`](../v22.2/monitor-and-debug-changefeeds.html#using-changefeed-metrics-labels) option. [#97509][#97509]
```

Comment:
supernit: `in order to use the ... option`

Reply:
yesss!